packages/mapviewer/src/utils/components/FileInput.vue

```diff
@@ -314,4 +329,12 @@ function onFileSelected(evt) {
 .local-file-input {
     cursor: pointer;
 }
+
+.warning-feedback {
+    display: block;
+    width: 100%;
+    margin-top: 0.25rem;
+    font-size: 0.875em;
+    color: $warning;
```

Comment:
please use Bootstrap classes instead of rebuilding new style each time. This will greatly help us when we will move away from Bootstrap (the less custom style we have, the better we are to start a migration)

Reply:
yeah, I did not retouch that after the rebase.